_posts/2023-06-26-Embracing-Kanban.md

```diff
@@ -6,11 +6,16 @@ categories: agile
 ---
 
 # Introduction
+In our quest to improve our workflow and ensuring a great product providing a better customer experience, we have made a significant decision to move away from traditional sprint planning and the old-fashioned waterfall. Instead, we have embraced Kanban, a methodology that gives us the flexibility we need to handle the demands of support scenarios. This change allows us to address urgent tasks promptly without being tied to strict sprint schedules, ultimately putting our customers at the forefront.
```

Comment:
We weren't following the `old-fashioned waterfall` model, so I'd remove that part.

Reply:
In the end of the day, we have a huge waterfall mindset behind the scenes, it is why I mentioned that, maybe I should have done it differently